modules/storage.js

```diff
@@ -0,0 +1,27 @@
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(target) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (`${book.id}` === target.id) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] Please change these functions to arrow functions to follow the ES6 syntax as requested in the project requirements.